tests/try/basic/test.sh

```diff
@@ -42,6 +42,13 @@ rlJournalStart
         rlAssertGrep "Run .* successfully finished. Bye for now!" $rlRun_LOG
     rlPhaseEnd
 
+    rlPhaseStartTest "Arch"
+        rlRun -s "./arch.exp"
+        rlAssertGrep "Let's try.*/plans/basic" $rlRun_LOG
+        rlAssertGrep "arch: aarch64" $rlRun_LOG
+        rlAssertGrep "Run .* successfully finished. Bye for now!" $rlRun_LOG
+    rlPhaseEnd
+
```

Comment:
The test is failing because it needs support for nested virtualization. One option could be to create a new `/tests/try/virtual` test, move this part and `exp` file there and tag it as `provision-only`. But I'm afraid, when executed under nested virtualization in Testing Farm it would be very slow. @thrix, @frantisekz, thoughts?

Reply:
It is slow, and as we talked about it, it should be rather checked via a unit test, which might be out of scope of this change. @happz promised to look at it afaik. For now I would just disble this test.

Reply:
As @happz promised and discussed in slack, I applied his suggestion and removed exp test